### PR TITLE
Feature/news letter signup footer

### PR DESCRIPTION
**Pull request by Njoe00**
adding news letter container for email sign up.

### CODE REVIEW

File: src/components/NewsLetterSignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+export default function NewsLetterSignUp() {
+  const [buttonColour, setButtonColour] = useState(false);
+
+  const HandleButtonColour = () => {
+    setButtonColour(!buttonColour);
+  };
```
Comment (MKCMMSK) [inline src/components/NewsLetterSignUp.tsx]: This is extremely unclear what this really does. when you setButtonColour, the expected behavior is that I'm setting a color to change to but this is a boolean. when it's a boolean the naming should reflect the binary nature of the action.

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
just a small change

File: src/components/NewsLetterSignUp.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/NewsLetterSignUp.tsx]: changed to changeColourButton

Comment (MKCMMSK) [reply src/components/NewsLetterSignUp.tsx]: That's still not good enough unfortunately. changeColourButton implies that I can change the color. 

generally when it comes to booleans, you would want a naming convension of "is___" to denote a true or false value.

Comment (Njoe00) [reply src/components/NewsLetterSignUp.tsx]: resolved.

**Pull request merged**